roles/common/tasks/container_mirror.yml

```diff
@@ -23,3 +23,10 @@
   copy:
     dest: "{{ container_mirror_cert_path }}/docker-mirror.crt"
     content: "{{ container_mirror_cert }}"
+
+- name: Install registries-conf-ctl 
+  pip:
+    name: git+https://github.com/sebastian-philipp/registries-conf-ctl 
+
+- name: Add local docker.io registry mirror
+  command: registries-conf-ctl add-mirror docker.io docker-mirror.front.sepia.ceph.com
```

Comment:
Is `:5000` required or is that assumed?

Reply:
I'd be super glad, if you could take over this PR. As I said, I have close to no expertise in ansible playbooks.

Reply:
May I suggest two tasks?

```
- name: Add local docker.io registry mirror | docker
  command: registries-conf-ctl --conf=/etc/docker/daemon.json add-mirror docker.io docker-mirror.front.sepia.ceph.com:5000
  when:
    - check_for_docker is success
    - check_for_podman is not success

- name: Add local docker.io registry mirror | podman
  command: registries-conf-ctl --conf=/etc/containers/registries.conf add-mirror docker.io docker-mirror.front.sepia.ceph.com:5000
  when:
    - check_for_docker is not success
    - check_for_podman is success
```

Reply:
support for docker is not yet implemented in registries-conf-ctl. let me have a look....

Reply:
> support for docker is not yet implemented in registries-conf-ctl. let me have a look....

If you can handle this portion, I'd be happy to handle the ceph-cm-ansible changes.

Reply:
https://github.com/sebastian-philipp/registries-conf-ctl#registries-conf-ctl now supports also the daemon.json. 

```
registries-conf-ctl add-mirror docker.io docker-mirror.front.sepia.ceph.com:5000
```

should now work for podman and docker

Reply:
How does your tool work?  I can't read python so I'm going to ask dumb questions...

If `docker` and `podman` commands are both detected, will the tool modify both config files?  What if neither are detected?  Does it care?  Will it create the files?

Reply:
guess it needs some more docs.

> If docker and podman commands are both detected, will the tool modify both config files?

yes

> What if neither are detected? 

It will fail. 

> Will it create the files? 

no